game/menu/README.md

```diff
@@ -0,0 +1,39 @@
+# Information for Contributers
+
+## GameManager
+The Gamemanager keeps track of all minigames and starts/stops them when needed.  
```

Comment:
```suggestion
## GameManager
The GameManager keeps track of all minigames and starts/stops them when needed.  
```
We should always use the same spelling when referring to a scene/class/singletone.
And they should be in PacalCase.